frontend/src/metabase/components/SaveQuestionModal.css

```diff
@@ -0,0 +1,23 @@
+:local(.saveQuestionModalFields) {
+    overflow: hidden;
+}
+
+.saveQuestionModalFields {
+    overflow: hidden;
```

Comment:
Good point. I might as well get rid of the :local class and use the plain CSS class for the element too since I need the plain CSS class for the `ReactCSSTransitionGroup` anyways.